fontforgeexe/fontinfo.c

```diff
@@ -196,6 +196,15 @@ static GTextInfo ibmfamily[] = {
     { (unichar_t *) N_("Sy Miscellaneous"), NULL, 0, 0, (void *) 0xc0f, NULL, 0, 0, 0, 0, 0, 0, 1, 0, 0, '\0'},
     GTEXTINFO_EMPTY
 };
+static GTextInfo stylemap[] = {
```

Comment:
I find the casting sort of confusing. How about putting an annotated set or array of integers before this and storing pointers into it? (I'm not saying that's the best approach.) There at least needs to be a comment describing what is happening.